tests/test_clustering.py

```diff
@@ -126,3 +129,106 @@ def test_clustering_no_edges(test_helpers, dialect):
     # due to blocking rules, df_predict will be empty
     df_predict = linker.inference.predict()
     linker.clustering.cluster_pairwise_predictions_at_threshold(df_predict, 0.95)
+
+@mark_with_dialects_excluding()
+@mark.parametrize(
```

Comment:
I think if we can reuse the internals.clustering.cluster_pairwise_predictions_at_multiple_thresholds function then we only need the most basic test here that it gives a sensible result when run on the `linker` - the tests for `internals.clustering.cluster_pairwise_predictions_at_multiple_thresholds` should handle the correctness of the algo